src/LineBot/Events/LineEvent.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace Line
 {
-    internal sealed class LineEvent : ILineEvent, IBeacon, IMessage, IPostback
+    public sealed class LineEvent : ILineEvent, IBeacon, IMessage, IPostback
```

Comment:
This class should remain internal.

Reply:
This is what I was talking about in the last commits review. 
--------+
Well If you see the latest commits, you'll see if `List<LineEvent>` is used then LineEvent must be made public and doing so will include a lot of documentation along with XML comments. So if you have any other solution to this problem, I can think about that.
![image](https://user-images.githubusercontent.com/33629087/48270060-2c53a100-e48d-11e8-94a4-b7321a03297b.png)